NAMESPACE

```diff
@@ -1,4 +1,6 @@
 # Generated by roxygen2: do not edit by hand
 
+import(terra)
```

Comment:
Did you add this statement to `NAMESPACE` by hand? I think it is better to include a line like 

```R
#' @import terra
```

in `R/topotoolboxr-package.R` so Roxygen automatically adds the `import(terra)` line to `NAMESPACE`.